modules/ivc_champva/app/jobs/ivc_champva/missing_form_status_job.rb

```
@@ -60,13 +58,27 @@ def construct_email_payload(form, template_id)
     #
     # @param form [IvcChampvaForm] form object in question
     # @param template_id [string] key for template to use in `IvcChampva::Email::EMAIL_TEMPLATE_MAP`
-    def send_failure_email(form, template_id)
+    # @param additional_context [hash] contains properties form_id and form_uuid
+    #   (e.g.: {form_id: '10-10d', form_uuid: '12345678-1234-5678-1234-567812345678'})
+    def send_failure_email(form, template_id, additional_context)
       form_data = construct_email_payload(form, template_id)
+
+      if Flipper.enabled?(:champva_vanotify_custom_callback, @current_user)
+        form_data = form_data.merge(
+          {
+            callback_klass: 'IvcChampva::ZsfEmailNotificationCallback',
+            callback_metadata: {
+              statsd_tag: 'veteran-ivc-champva-forms',
+              additional_context:
+            }
+          }
+        )
+      end
```

Passing this callback class to VA Notify when we send the email allows us to gather the email delivery status, which is then going to the DD silent failures dashboard.